CONTRIBUTING.md

```diff
@@ -337,6 +337,10 @@ The exercise should consist of, at minimum:
 * A test suite
 * A reference solution that passes the test (see [#reference-solution](#reference-solution))
 
+You will need to add the exercise to `"exercises"` section of the `config.json` file in the track.
+The order in which the exercises are listed there is the order in which they are fetched by default by `exercism fetch`.
+Typically, exercises are ordered by difficulty, unless there is a particular reason to do otherwise.
```

Comment:
tentatively applying choice 3, since I also think it seems good to document what's typical, and since I think the main concern is with any tracks that wish to do differently, and if you like 3 I'll hope any others will as well.